### PR TITLE
Fix resolver construction

### PR DESCRIPTION
**Pull request by simonovic86**

### CODE REVIEW

File: packages/http-client/src/ceramic-http-client.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default class CeramicClient implements CeramicApi {
     const keyDidResolver = KeyDidResolver.getResolver()
     const threeIdResolver = ThreeIdResolver.getResolver(this)
     this.context.resolver = new Resolver({
-      ...config.didResolver, ...threeIdResolver, ...keyDidResolver,
+      ...this._config.didResolver, ...threeIdResolver, ...keyDidResolver,
```
Comment (oed) [inline packages/http-client/src/ceramic-http-client.ts:59]: The config variable should be available here.

Comment (oed) [reply packages/http-client/src/ceramic-http-client.ts:59]: It's passed to the constructor

Comment (oed) [reply packages/http-client/src/ceramic-http-client.ts:59]: Oh, is it if it's unset?

**Review by oed — APPROVED** [review]
I see now. Makes sense 👍

File: packages/http-client/src/ceramic-http-client.ts (hunk 1 above)
Comment (simonovic86) [reply packages/http-client/src/ceramic-http-client.ts:59]: yep 🙂

**Pull request merged**